src/Vixen.Modules/Editor/FixtureGraphics/OpenGL/MovingHeadOpenGL.cs

```diff
@@ -671,9 +671,9 @@ public void Initialize(double length, double maxBeamLength, double beamTranspare
 					(float)-_geometry.GetBaseDepth());
 				
 				// Draw the text
-				// TODO .NET 6 commented _qFontDrawing.Print(_qFont, MovingHead.Legend, positionOfText, QFontAlignment.Left, MovingHead.LegendColor);
+				_qFontDrawing.Print(_qFont, MovingHead.Legend, positionOfText, QFontAlignment.Left, MovingHead.LegendColor);
 				_qFontDrawing.RefreshBuffers();
-				// TODO .NET 6 commented _qFontDrawing.ProjectionMatrix = viewMatrix * projectionMatrix;				
+				_qFontDrawing.ProjectionMatrix = viewMatrix * projectionMatrix;				
 				_qFontDrawing.Draw();
 			}
 		}
```

Comment:
Did you also look at line 703 QFontDrawing.DisposeStaticState() that I commented also? If it is not needed then we can remove the line.

Reply:
I don't see an updated PR yet.